quarto-style.scss

```diff
@@ -1857,11 +1871,6 @@ btn.action-button:hover {
   border: none;
 }
 
-/* improve position of code copy button */
```

Comment:
This was added during the Express rebuild, but it seems to have removed the code-copy icons from the website.